Tests/ForageSDKTests/ForageSDKTests.swift

```diff
@@ -34,6 +34,19 @@ final class ForageSDKTests: XCTestCase {
 
         XCTAssertEqual(MockForageSDK.shared.sessionToken, "dev_authToken123")
     }
+    
+    func testSetup_shouldUpdateSessionTokenAndMerchantID() {
```

Comment:
TODO: Add a test of setting the config, reading `ForageSDK.shared.environment`, then setting the config again.